adapters/http-vertx-base/src/main/java/org/eclipse/hono/adapter/http/AbstractVertxBasedHttpProtocolAdapter.java

```diff
@@ -975,29 +1014,45 @@ protected void setNonEmptyResponsePayload(final HttpServerResponse response, fin
                     log.debug("command for device has already fired [tenantId: {}, deviceId: {}]",
                             tenantObject.getTenantId(), deviceId);
                     commandContext.release();
-                } else {
-                    checkMessageLimit(tenantObject, command.getPayloadSize())
-                            .setHandler(result -> {
-                                if (result.succeeded()) {
-                                    addMicrometerSample(commandContext, commandSample);
-                                    // put command context to routing context and notify
-                                    ctx.put(CommandContext.KEY_COMMAND_CONTEXT, commandContext);
-                                } else {
-                                    // issue credit so that application(s) can send the next command
-                                    commandContext.reject(getErrorCondition(result.cause()), 1);
-                                    metrics.reportCommand(
-                                            command.isOneWay() ? Direction.ONE_WAY : Direction.REQUEST,
-                                            tenantObject.getTenantId(),
-                                            tenantObject,
-                                            ProcessingOutcome.from(result.cause()),
-                                            command.getPayloadSize(),
-                                            commandSample);
-                                }
-                                cancelCommandReceptionTimer(ctx);
-                                setTtdStatus(ctx, TtdStatus.COMMAND);
-                                responseReady.tryComplete();
-                            });
                 }
+
+//                if (requestProcessed.get()) {
```

Comment:
Do we need this huge block of commented out code?

Reply:
definitely not :+1: